msg.go

```diff
@@ -100,7 +100,7 @@ var TypeToString = map[uint16]string{
 	TypeANY:        "ANY", // Meta RR
 	TypeATMA:       "ATMA",
 	TypeAXFR:       "AXFR", // Meta RR
-	TypeCAA:        "CAA",
+	TypeCAA:        "TYPE257",
```

Comment:
This makes go-dns print TYPE257 everywhere it outputs CAA record as string and also messes up parsing of record when CAA used, will only understand TYPE257 (since StringToType that tokenizer uses is a reverse of TypeToString).